EnemyCollision.cs

```diff
@@ -14,7 +14,9 @@ void Start()
     // Update is called once per frame
     void Update()
     {
-        
+         Movement();
```

Comment:
Good job :) To keep the code clean, we can select the code and right click and select `Format`